homeassistant/components/duckdns/__init__.py

```diff
@@ -40,10 +40,11 @@
 SERVICE_TXT_SCHEMA = vol.Schema({vol.Required(ATTR_TXT): vol.Any(None, cv.string)})
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
If you remove this, what triggers the import of yaml to a config entry? Have you actually tested the change, and it works?

Reply:
You are right, I've pushed an update with the fix.